Frontend/validation/Item.schema.json

```diff
@@ -140,6 +140,15 @@
           }
         },
         "description": "Item properties/settings"
+      },
+      "itemJobTypes": {
+        "type": "array",
+        "items": {
+          "type": "string",
+          "enum": ["getData", "storeData", "prepareData", "analyzeAndTrainData", "trackData", "visualizeData", "develop", "generalTask", "others"]
+        },
+        "maxItems": 2,
```

Comment:
I think it’s better to have the same logic as in the backend, where there is a dedicated validator for this. If you decide to keep it as is, it would be better to use the same error message, as the current approach will generate a schema-related message, while we have our own pre-defined message

Reply:
I see.
I agree it's better to be aligned.
Since I already have this change in the stage and main branches without a validator, and it was tested, I'd rather not change it for production.

I will publish PRs for our internal repos and will update this one after it is tested.